src/mono/mono/mini/interp/transform.c

```diff
@@ -8587,6 +8581,13 @@ interp_cprop (TransformData *td)
 				local_defs [dreg].def_index = ins_index;
 			}
 
+			// We always store to the full i4, except as part of STIND opcodes. These opcodes can be
+			// applied to a local var only if that var has LDLOCA applied to it
+			if ((opcode >= MINT_MOV_I1 && opcode <= MINT_MOV_U2) && !td->locals [sregs [0]].indirects) {
```

Comment:
Should it be 'not applied to it' ?

Reply:
I was refering to the STIND opcodes, in the sense that they can store to a local var only if the local var has its address taken.